tools/lint/COPYING

```diff
@@ -0,0 +1,38 @@
+This test suite comes from
+https://github.com/w3c/wpt-tools/
+with modification:
+
+lint.py
+
+ -def all_git_paths():
```

Comment:
No need to detail the modification, just summary your update with brief sentence.